plugins/modules/remote_management/redfish/ilo_redfish_config.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021-2022 Hewlett Packard Enterprise, Inc. All rights reserved.
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ilo_redfish_config
+short_description: Sets or updates a configuration attribute. For use with HPE iLO operations that require Redfish OEM extensions.
+version_added: 4.1.0
+description:
+    - "Provides an interface to manage configuration attributes."
+options:
+  category:
+    required: true
+    type: str
+    description:
+      - Command category to execute on iLO
+    choices: ['Manager', 'Sessions']
+  command:
+    required: true
+    description:
+      - List of commands to execute on iLO
```

Comment:
TBH this looks like it belongs to a `ilo_redfish_command` module, not like a `ilo_redfish_config` module. Why do you allow to execute arbitrary commands?

Also how do you pass parameters for multiple of these commands?

Reply:
The functions being executed in this module are a list of manager configurations and hence it's in ilo_redfish_config.

We pass only attribute_name and atribute_value for one command at a time.